samples/apps/copilot-chat-app/SKWebApi/appsettings.json

```diff
@@ -1,48 +1,42 @@
-    "Endpoint": "[Enter the endpoint of your Azure OpenAI instance, e.g. https://contoso.openai.azure.com]"
-    /* Use user secrets or Key Vault - Putting secrets in a plain-text file
-       is a security risk and should be done as a last resort only. */
-    //"Key": "[DO NOT CHECK IN - Key to your (Azure) OpenAI instance.]"
+    "AIService": "AzureOpenAI", // Or "OpenAI" when using OpenAI directly
+    "DeploymentOrModelId": "text-davinci-003", // Azure OpenAI deployment name or OpenAI model to use
+    "Endpoint": "" // Your Azure OpenAI endpoint (e.g. https://contoso.openai.azure.com) - Leave blank when using OpenAI directly
+    /* "Key": Key for your (Azure) OpenAI instance. Use user secrets, environment variable or Key Vault to populate.
+       Putting secrets in a plain-text file is a security risk and should be done as a last resort only.
+       Ex: dotnet user-secrets set "CompletionConfig:Key" "MY_COMPLETION_KEY" */
   },
+
   // Remove or comment out EmbeddingConfig if no embedding backend is required
   "EmbeddingConfig": {
     "Label": "Embeddings",
-    "AIService": "[Enter 'AzureOpenAI', or 'OpenAI']",
-    "DeploymentOrModelId": "[Enter the Azure OpenAI deployment name or OpenAI model, e.g. 'text-embedding-ada-002']",
-    "Endpoint": "[Enter the endpoint of your Azure OpenAI instance, e.g. https://contoso.openai.azure.com]"
-    /* Use user secrets or Key Vault - Putting secrets in a plain-text file
-       is a security risk and should be done as a last resort only. */
-    //"Key": "[DO NOT CHECK IN - Key to your (Azure) OpenAI instance.]"
-  },
-  /* The following identity settings need to be configured
-     before the project can be successfully executed.
-     For more info see https://aka.ms/dotnet-template-ms-identity-platform */
-  "AzureAd": {
-    "Instance": "https://login.microsoftonline.com/", // Or whichever instance you use
-    "Domain": "[Enter the domain of your tenant, e.g. contoso.onmicrosoft.com]",
-    "TenantId": "[Enter 'common', or 'organizations' or the Tenant Id (Obtained from the Azure portal. Select 'Endpoints' from the 'App registrations' blade and use the GUID in any of the URLs), e.g. da41245a5-11b3-996c-00a8-4d99re19f292]",
-    "ClientId": "[Enter the Client Id (Application ID obtained from the Azure portal), e.g. ba74781c2-53c2-442a-97c2-3d60re42f403]",
-    "Scopes": "access_as_user",
-    "CallbackPath": "/signin-oidc"
+    "AIService": "AzureOpenAI", // Or "OpenAI" when using OpenAI directly
+    "DeploymentOrModelId": "text-embedding-ada-002", // Azure OpenAI deployment name or OpenAI model to use
+    "Endpoint": "" // Your Azure OpenAI endpoint (e.g. https://contoso.openai.azure.com) - Leave blank when using OpenAI directly
+    /* "Key": Key for your (Azure) OpenAI instance. Use user secrets, environment variable or Key Vault to populate.
+       Putting secrets in a plain-text file is a security risk and should be done as a last resort only.
```

Comment:
`Putting secrets in a plain-text file is unsafe and not recommended.`

Reply:
Changed